libs/angular-testing/src/lib/harnesses/menu-item-with-icon.harness.ts

```diff
@@ -0,0 +1,12 @@
+import { MenuItemHarness } from './menu-item.harness'
+
+export class MenuItemWithIconHarness extends MenuItemHarness {
+  async hasIcon(icon: string): Promise<boolean | undefined> {
+    const classList = await (await this.locatorForOptional('i')())?.getAttribute('class')
+    return classList?.includes(icon)
+  }
+
+  async isExternal(): Promise<boolean | undefined> {
+    return (await (await this.host()).getAttribute('ng-reflect-router-link')) === null
```

Comment:
Please do not use the ng-attribute.
Please try to find an alternative

Reply:
Seems like there is no information that could replace this.

I'll try to find out if there is a way to just click on the item and create expectation for correct routing

Reply:
Alright. Thank you. Otherwise please align with Jan on monday how to proceed